components/web/apps/org.wso2.carbon.apimgt.publisher/pom.xml

```diff
@@ -79,5 +163,10 @@
             </plugin>
         </plugins>
     </build>
-
+    <properties>
+        <maven.test.skip>false</maven.test.skip>
+        <npm.executable>npm</npm.executable>
```

Comment:
You can use a maven activation to change this npm.executable property to `npm.cmd` on windows. Otherwise the build wouldn't work on windows machines. Example: https://github.com/ballerinalang/composer/blob/master/modules/web/pom.xml#L149